psycopg/psycopg/_cursor_base.py

```diff
@@ -334,7 +334,7 @@ def _stream_send_gen(
         yield from send(self._pgconn)
 
     def _stream_fetchone_gen(self, first: bool) -> PQGen[PGresult | None]:
-        res = yield from fetch(self._pgconn)
+        res: PGresult | None = yield from fetch(self._pgconn)
         if res is None:
             return None
```

Comment:
Maybe just `return res` is enough?

Reply:
No, the error is downstream, if the annotation is misisng:

```
psycopg/psycopg/_cursor_base.py:347: error: Returning Any from function
declared to return "PGresult | None"  [no-any-return]
                return res
                ^~~~~~~~~~
Found 1 error in 1 file (checked 1 source file)
```